models/demos/tg/llama3_70b/tt/llama_common.py

```diff
@@ -82,3 +82,44 @@ def tt_sharded_all_gather(input_tensor, mesh_device, cluster_axis, dim, num_link
         mesh_device=mesh_device,
         memory_config=memory_config,
     )
+
+
+def tt_sharded_distributed_rmsnorm(mesh_device, inp, epsilon, gamma):
```

Comment:
Not big of a deal but I would recommend putting inp (main inputs of method) as first argument instead of mesh_device.